src/async-home.js

```diff
@@ -17,7 +23,10 @@ const assignmentsInterval = async () => {
 };
 
 const writeChecks = function () {
-  let completedID = [];
+  console.log(schoologyData);
+  console.log(completeColor);
+
+  let completedEle = [];
```

Comment:
```suggestion
  const completedAssignments = [];
```
I would make the name more clear, and also you are using `let` way more than you need to. A rule of thumb in javascript is to use `const` unless you are specifically RE-assigning that variable in the future.